galvani/BioLogic.py

```diff
@@ -494,6 +540,8 @@ def __init__(self, file_or_path):
             raise ValueError("Invalid magic for .mpr file: %s" % magic)
 
         modules = list(read_VMP_modules(mpr_file))
+        mpr_file.close()  # close the mpr.file after reading all of it
```

Comment:
I added the mpr_file.close() here, it seemed to me it was "cleaner" to close the file when finished reading it

Reply:
I would rather avoid that - if the function takes a file object then it should not close it.

Reply:
removed